api/features/serializers.py

```diff
@@ -11,7 +11,12 @@
     HideSensitiveFieldsSerializerMixin,
 )
 from projects.models import Project
-from users.serializers import UserIdsSerializer, UserListSerializer
+from users.models import UserPermissionGroup
+from users.serializers import (
+    MyUserPermissionGroupsSerializer,
```

Comment:
Love the reuse of this serializer, but I think we should probably find a more generic name for the serializer. Alternatively, we could do an `import MyUserPermissionGroupsSerializer as ...` but I'd rather just rename the original serializer.

Reply:
Good idea! I'll rename the serializer to be more generic